File: backend/src/main/java/com/safetunnel/backend/vpn/server/VpnServerResponse.java
package com.safetunnel.backend.vpn.server;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class VpnServerResponse {
    private String id;
    private String name;
    private String country;
    private String city;
    private String host;
    private int port;
    private String publicKey;
    private String status;
    private Integer loadPercentage;
}

File: backend/src/main/java/com/safetunnel/backend/vpn/server/VpnServerResponse.kt
package com.safetunnel.backend.vpn.server

data class VpnServerResponse(
    val id: String,
    val name: String,
    val country: String,
    val city: String,
    val host: String,
    val port: Int,
    val publicKey: String,
    val status: String,
    val loadPercentage: Int?
)